clients/fides-js/src/lib/cookie.ts

```diff
@@ -270,11 +330,16 @@ export const updateExperienceFromCookieConsentNotices = ({
       return { ...notice, current_preference: preference };
     });
 
+  const updatedPrefetchedExperience = {
+    ...experience,
+    privacy_notices: noticesWithConsent,
+  };
+
   fidesDebugger(
     `Returning updated pre-fetched experience with user consent.`,
-    experience,
+    updatedPrefetchedExperience,
```

Comment:
Before we were logging the previous experience but I've updated to include the updated info